examples/multithreaded.rs

```diff
@@ -19,6 +19,8 @@ fn main() {
             .with_inner_size(WINDOW_SIZE.into())
             .build(&event_loop)
             .unwrap();
+        let video_modes: Vec<_> = window.current_monitor().video_modes().collect();
```

Comment:
What when they move it between monitors....

Reply:
The reason this is here in the first place is so that we can cycle between video modes with the arrow keys. The window will be placed fullscreen on the monitor it first appeared on. Video modes can differ between monitors and tracking this per monitor seems like it would unnecessarily complicate this example.

Reply:
There is no additional tracking really required. Look at my glutin example. Just do this call every time before it is needed. Before calling fullscreen print out the one which is actually chosen.

Yes, that means the one that gets picked in the end might be slightly different then the one they wanted if they move screens after selecting, but that strikes me as better then their window jumping to a completely different window.

I don't want to be debugging only to have the window fly to my main screen, which I like to put my code on.

If that really concerns you, we can on every `WindowEvent::Moved` event check if the current window has changed, and if it has just update `video_modes` and print out the new video mode. That would be the best solution IMHO.

Reply:
Done!